#!/usr/bin/env python
PACKAGE = "bobi_vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("h_low",    double_t,    0, "H low threshold", 0.55, 0, 255)
gen.add("s_low",    double_t,    0, "S low threshold", 0.55, 0, 255)
gen.add("v_low",   double_t,   0, "V low threshold",  0.55, 0, 255)

gen.add("h_high",    double_t,    0, "H high threshold", 1.6, 0, 255)
gen.add("s_high",    double_t,    0, "S high threshold", 1.6, 0, 255)
gen.add("v_high",   double_t,   0, "V high threshold",  1.0, 0, 255)


exit(gen.generate(PACKAGE, "robot_tracking_node", "ColourDetector"))